src/reducers/calendarReducer.js

```diff
@@ -13,32 +12,26 @@ export default function Calendar(state = initialState.CalendarData, { payload, t
 
         case actions.DELETE_EVENT:
             const removeIndex = newState.events_in_view.map((item) => { return item.id; }).indexOf(payload.id);
-            newState.events_in_view.splice(removeIndex, 1)
+            newState.events_in_view.splice(removeIndex, 1);
             return newState;
 
-
         case actions.DELETE_ALL_EVENTS:
             return state;
 
-
         case actions.FILTER_EVENT:
-
-
-            const filter_key = payload.key
-            const filter_value = payload.value
-
+            const filter_key = payload.key;
+            const filter_value = payload.value;
 
             // Search by subject
             newState.events_in_view.filter((allCourse) => {
-                return allCourse[filter_key] === filter_value
+                return allCourse[filter_key] === filter_value;
             })
```

Comment:
`filter` parameter can be simplified to `(allCourse) => allCourse[filter_key] === filter_value`

Reply:
How did this even show up as a change 😶